src/vos/vos_gc.c

```diff
@@ -29,7 +29,7 @@ enum {
  * - each item consumes 16 bytes, 250 * 16 = 4000 bytes
  * - together is 4080 bytes, reserve 16 bytes for future use
  */
-static int gc_bag_size	= 250;
+static int gc_bag_size	= 250 + 3 * 256;
```

Comment:
What's the change for? It's not mentioned in commit message.

Reply:
This change was recommended by Liang so can have more entries in each bag. I'll update the message.